requirements/conda.txt

```diff
@@ -16,7 +16,7 @@ pyxdg
 pyzmq
 qdarkstyle >=2.7
 qtawesome >=0.5.7
-qtconsole >=4.5.2
+qtconsole >=4.5.5
```

Comment:
Please revert this change because it's not necessary given that we're pulling qtconsole from Github and we also need to wait until Anaconda releases packages for this version.